packages/server/resources/views/emails/email_verification.edge

```diff
@@ -0,0 +1,21 @@
+<mjml>
+  <mj-body>
+    <mj-section>
+      <mj-column>
+
+        <mj-text align="left" color="#F45E43" font-family="Arial, sans-serif" font-size="13px" line-height="22px" padding-bottom="5px" padding-top="25px" padding="10px 25px">
+          <p style="line-height: 60px; text-align: center; margin: 10px 0;font-size:55px;color:#212121;font-family:'Times New Roman',Helvetica,Arial,sans-serif"><b>Musicroom</b></p>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I do not know MJML. Why do you use a `p` tag to display text here, with inline styles, while below you directly put text in a `mj-text` component?

Reply:
I've been using a template centered text section for the token and title